src/v2/cookbook/creating-custom-scroll-directives.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Creating Custom Scroll Directives
+type: cookbook
+order: 5
+---
+
+## Simple Example
+
+There are many times that we might want to add a bit of behavior, especially animation, to a scroll event on a site. There are many ways to do so, but the path with the least amount of code and dependencies is to use a [custom directive](https://vuejs.org/v2/guide/custom-directive.html) to create a hook for anything that fires off a particular scroll event.
+
+``` js
+Vue.directive('scroll', {
+  inserted: function(el, binding) {
```

Comment:
Call me nick-picky, but there should be a space after `function` following to Vue's coding standard.

Reply:
Oh, and often we don't use semi-colons ;) (pun not intended).